client/gutenberg/extensions/tiled-gallery/view.js

```diff
@@ -50,25 +49,31 @@ const resizeGallery = ( { galleryNode, width, columns, layout } ) => {
 };
 
 /**
- * Throttled resize event
  * Compares gallery width to its previous width to decide if to resize it.
+ * @param {Array<ResizeObserverEntry>} entries Resized entries
  */
-const throttleOnResize = throttle( entries => {
-	for ( const entry of entries ) {
-		const { width } = entry.contentRect;
-		// Don't resize if width didn't chance
-		if ( width !== entry.target.getAttribute( 'data-width' ) ) {
-			// Store width for later comparison
-			entry.target.setAttribute( 'data-width', width );
-			resizeGallery( {
-				columns: parseInt( entry.target.getAttribute( 'data-columns' ), 10 ),
-				galleryNode: entry.target,
-				layout: getActiveStyleName( entry.target.className ),
-				width,
-			} );
-		}
+function handleResize( entries ) {
```

Comment:
I would really like to get all the resize handling using the same code.

Reply:
Agreed, that would be good 👍